test/videojs-contrib-hls.test.js

```diff
@@ -811,6 +823,10 @@ QUnit.test('uses the lowest bitrate if no other is suitable', function(assert) {
 
   // the lowest bitrate playlist is much greater than 1b/s
   this.player.tech_.hls.bandwidth = 1;
+  this.player.tech_.hls.playlists.master.playlists[0].attributes.BANDWIDTH = 9e10;
```

Comment:
Since the selector now uses a moving average instead of the instantaneous, the bandwidth value that is being compared is the moving average of 1 and 4Mb, instead of 1. Needed to update the bandwidth of the playlist so it would still select